internal/cmd/coverage.go

```diff
@@ -69,7 +72,7 @@ func NewCoverageCommand(config *cnf.Config, logger *logrus.Logger) *cobra.Comman
 			g, ctx := errgroup.WithContext(cmd.Context())
 			g.Go(func() error {
 				var provider cache.Graphite
-				provider, err := graphite.New(config.Graphite.URL, &http.Client{Timeout: time.Second}, logger)
+				provider, err := graphite.New(config.Graphite.URL, &http.Client{Timeout: 5 * time.Second}, logger, prg)
```

Comment:
> Timeout: 5 * time.Second
I think the timeout should be configurable. I faced with a problem, that the default timeout wasn't enough.

Reply:
Yep, I completely agree with you.

Reply:
Feel free to open an issue related to your suggestion.

Reply:
Fixed in https://github.com/kamilsk/grafaman/pull/49